### PR TITLE
Components

### PR DESCRIPTION
**Pull request by mbsyaswanth**

### CODE REVIEW

File: src/components/GitApp/SplashScreen/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+
+import {Actions, ActionConst} from 'react-native-router-flux';
+import {Container, Logo} from './styledComponents';
+
+class SplashScreen extends Component {
+  componentDidMount() {
+    setTimeout(() => {
+      Actions.users({type: ActionConst.REPLACE});
```
Comment (0xc0d3r) [inline src/components/GitApp/SplashScreen/index.js]: write this in `NavigationUtils`

File: src/components/GitApp/SplashScreen/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    }, 1500);
+  }
+
+  render() {
+    return (
+      <Container>
+        <Logo> Git Explorer </Logo>
```
Comment (0xc0d3r) [inline src/components/GitApp/SplashScreen/index.js]: Maintain it in I18n files

File: src/components/GitApp/UsersList/User/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  render() {
+    return (
+      <TouchableNativeFeedback data-testid="user" onPress={() => goToRepos()}>
+        <UserContainer>
+          <Avatar
+            source={{
+              uri: 'https://facebook.github.io/react-native/img/tiny_logo.png',
+            }}
+          />
+          <UserText> User </UserText>
```
Comment (0xc0d3r) [inline src/components/GitApp/UsersList/User/index.js]: I18n text

**Review by 0xc0d3r — CHANGES_REQUESTED** [review]

File: src/components/GitApp/UsersList/User/index.js (hunk 3 above)
Comment (mbsyaswanth) [reply src/components/GitApp/UsersList/User/index.js]: added

File: src/components/GitApp/SplashScreen/index.js (hunk 1 above)
Comment (mbsyaswanth) [reply src/components/GitApp/SplashScreen/index.js]: utils
 - navigation
    - index.js

**Review by 0xc0d3r — APPROVED** [review]

**Pull request merged**